### ISSUE TITLE
plone.app.toolbar contents not loading in child site

### ISSUE THREAD

**Issue by rnunez80** [collective/collective.lineage#22]
i'm using plone.app.toolbar 1.4.0 when you click on content it does not load the content of the folder

plone.app.toolbar is looking for @@getVocabulary in the child site but gets a 404

**Comment by rnunez80** [comment]
you can circumvent this bug if you are using nginx by writing a location 
```
location ~ ^/(.*)/(.*)@@getVocabulary$ {rewrite  ^(.*)/@@getVocabulary$  https://www.york.cuny.edu/@@getVocabulary?query=%7B%22criteria%22%3A%5B%7B%22i%22%3A%22path%22%2C%22o%22%3A%22plone.app.querystring.operation.string.path%22%2C%22v%22%3A%22$1%3A%3A1%22%7D%5D%2C%22sort_on%22%3A%22getObjPositionInParent%22%2C%22sort_order%22%3A%22ascending%22%7D&name=$arg_name&batch=$arg_batch&attributes=$arg_attributes?  permanent;}
	location ~ ^/(.*)/(.*)@@qsOptions$ {rewrite  ^(.*)$  https://www.york.cuny.edu/@@qsOptions  permanent;}
```

**Comment by thet** [comment]
@rnunez80 are you using plone.app.toolbar in Plone 4? Since Plone 5 has it's own toolbar and I don't remember this bug beeing present there, I think your bug is a rare corner case.

**Comment by thet** [comment]
@rnunez80 you're using plone.app.widgets on Plone 4, right?
Your problem should be fixed here: https://github.com/plone/plone.app.widgets/pull/101
Only, this version of plone.app.widgets is not released yet. another issue has to be resolved first, before we can make a release ( https://github.com/plone/plone.app.widgets/commit/3f3dc690442970b1e033629573aeff568d7e9262 related to https://github.com/plone/plone.app.z3cform/pull/34 ). Meanwhile, just use the source checkout.
